doc/seg-schema.json

```diff
@@ -1,198 +1,201 @@
 {
-  "@context": "http://qiicr.org/dcmqi/contexts/dcmqi.jsonld",
-  "id": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/seg-schema.json#",
-  "$schema": "http://json-schema.org/draft-04/schema#",
-  "properties": {
-    "seriesAttributes": {
-      "type": "object",
-      "properties": {
-        "ContentCreatorName": {
-          "allOf": [
+  "@context":"http://qiicr.org/dcmqi/contexts/dcmqi.jsonld",
+  "id":"https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/seg-schema.json#",
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "properties":{
+    "seriesAttributes":{
+      "type":"object",
+      "properties":{
+        "ContentCreatorName":{
+          "allOf":[
             {
-              "$ref": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/PN"
+              "$ref":"https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/PN"
```

Comment:
@fedorov Why are you removing all those spaces? It's general json syntax to have 
```
"key": "value" 
```

Even when you use a library for generating JSON it always comes out that way. Maybe it's a matter of taste but let's try keep it one way. 

http://www.jsonschemavalidator.net/ also formats the way I mentioned.